main.go

```diff
@@ -72,54 +85,99 @@ func addGriefer(s *discordgo.Session, m *discordgo.MessageCreate) {
 				for _, grief := range griefers {
 					myGriefees = append(myGriefees, fmt.Sprintf("<@%s>", grief))
 				}
-				s.ChannelMessageSend(m.ChannelID, strings.Join(myGriefees, " "))
+				sess.ChannelMessageSend(mess.ChannelID, strings.Join(myGriefees, " "))
 
 				return
 			}
 		}
 
-		for _, mention := range m.Mentions {
+		for _, mention := range mess.Mentions {
 			griefers = append(griefers, mention.ID)
 		}
 
-		s.ChannelMessageSend(m.ChannelID, "This brotha is getting griefed")
+		sess.ChannelMessageSend(mess.ChannelID, "This brotha is getting griefed")
 
 		return
 	}
 }
 
-func handleCommands(s *discordgo.Session, m *discordgo.MessageCreate) {
-	if m.Author.ID != s.State.User.ID {
-		log.Printf(m.Author.Username + ": " + m.Content)
+func handleCommands(sess *discordgo.Session, mess *discordgo.MessageCreate) {
+	if mess.Author.ID != sess.State.User.ID {
+		log.Printf(mess.Author.Username + ": " + mess.Content)
 	}
 
 	// Respond to messages
-	if m.Content == "!hello" {
+	if mess.Content == "!hello" {
 		// Reply with a message
-		s.ChannelMessageSend(m.ChannelID, "Hello, "+m.Author.Username+"!")
+		sess.ChannelMessageSend(mess.ChannelID, "Hello, "+mess.Author.Username+"!")
 	}
 
-	if m.Content == fmt.Sprintf("<@%v>", vetroId) {
-		s.ChannelMessageSend(m.ChannelID, "Hey, "+m.Author.Username+"!")
+	if mess.Content == fmt.Sprintf("<@%v>", vetroId) {
+		sess.ChannelMessageSend(mess.ChannelID, "Hey, "+mess.Author.Username+"!")
 
 	}
-	if m.Content == "https://imgur.com/a/XQ3pPTQ" {
-		s.ChannelMessageSend(m.ChannelID, "Assemble!!!!!")
+	if mess.Content == "https://imgur.com/a/XQ3pPTQ" {
+		sess.ChannelMessageSend(mess.ChannelID, "Assemble!!!!!")
 	}
 }
 
-func handleVoiceStateUpdate(s *discordgo.Session, m *discordgo.VoiceStateUpdate) {
-	if m.ChannelID != channelId {
+func handleVoiceStateUpdate(sess *discordgo.Session, mess *discordgo.VoiceStateUpdate) {
+	if mess.ChannelID != channelId {
 		return
 	}
 
-	if m.VoiceState.UserID == morthisId {
-		s.ChannelMessageSend(m.ChannelID, fmt.Sprintf("Hello gaylord <@%s>", morthisId))
+	if mess.VoiceState.UserID == morthisId {
+		sess.ChannelMessageSend(mess.ChannelID, fmt.Sprintf("Hello gaylord <@%s>", morthisId))
 	}
 
 	for _, griefee := range griefers {
-		if m.VoiceState.UserID == griefee {
-			s.GuildMemberMove(m.GuildID, griefee, &channelId)
+		if mess.VoiceState.UserID == griefee {
+			sess.GuildMemberMove(mess.GuildID, griefee, &channelId)
+		}
+	}
+}
+
+func birthdateCheck(sess *discordgo.Session) { 
+	var birthDateDiscId int
+	currDate := fmt.Sprintf(time.Now().Format("2006-01-02"))
+        targetTime := time.Date(now.Year(), now.Month(), now.Day(), 15, 0, 0, 0, now.Location()) 
+        if now.Hour() == targetTime.Hour() { 
+		birthDateDiscId = birthdateQuery(currDate)
+	}
+	birthDateDiscId = birthdateQuery(currDate)
+	if birthDateDiscId != -1 {
```

Comment:
Generally its a good idea to limit nesting as much as possible, and a function like this is a perfect example of something that can be slightly adjusted to improve the quality / ease of understanding:

```go
if birthDateDiscId == -1 {
  return
}
```